### ISSUE TITLE
A Question About Face Mask Inpainting

### ISSUE THREAD

**Issue by francqz31** [pq-yang/PGDiff#2]
Hello Author , 

Have you compared the face inpainting of this paper to (Repaint: https://github.com/andreas128/RePaint) , I believe Repaint is training-free method which means no further training or fine-tuning is required.

Also can the face inpainting method here get finetuned solely for face/mask inpainting for better results ?

**Comment by pq-yang** [comment]
Hi @francqz31 , thank you for your attention and patience!

Similar to Repaint, our PGDiff framework doesn't require any training for the inpainting task, but we achieve this goal using a different strategy. We will provide a visual comparison with Repaint later.

As for how to finetune our PGDiff framework to improve inpainting results, it could be a promising idea that requires further exploration in future works.
